packages/core/src/utils/styleUtils/ssrStyles.ts

```diff
@@ -166,6 +169,8 @@ export const detachExperienceStyles = (experience: Experience): string | undefin
           // and this is where the over-writes for the default values are stored
           // yes, I know, it's a bit confusing
           componentVariablesOverwrites: currentNode.variables,
+          // and this is where the pattern node
```

Comment:
What is this comment supposed to express? Maybe adjust or just remove it :)

```suggestion
          // pass top-level pattern node to store instance-specific child styles for rendering
```